dataproxy/vercel-edge-functions/middleware.ts

```diff
@@ -0,0 +1,17 @@
+import type { NextRequest } from 'next/server'
+import { NextResponse } from 'next/server'
+import { PrismaClient } from '@prisma/client/edge'
+
+export async function middleware(request: NextRequest) {
+  const prisma = new PrismaClient({})
+
+  // we use a transaction as it has caused issues with tracing
+  // `traceparent: undefined` was being passed to the engine
+  // but we ultimately want to test that queries are sent well
```

Comment:
?

Reply:
It's an old comment from copy pasted code, it's not relevant anymore.